### ISSUE TITLE
support: Hugo version v0.139.5 is not supported

### ISSUE THREAD

**Issue by eskopp** [peaceiris/actions-hugo#662]
### Checklist

- [X] I am using the latest version of this action.
- [X] I have read the latest README and followed the instructions.
- [X] I have read the latest GitHub Actions official documentation and learned the basic spec and concepts.

### Describe your question

A new Hugo version was released yesterday. This cannot be installed via the flag latest. The YML code and the log are attached. 

# YML File
```yml
      - name: Install Hugo Extended
        uses: peaceiris/actions-hugo@v3
        with:
          hugo-version: "latest" # <-- error 404
          extended: true
```
# GitHub Action
![image](https://github.com/user-attachments/assets/e1530ccd-6804-49ac-a972-ea6b0503bb04)

### Relevant links

```markdown
Public repository: https://github.com/eskopp/AdventofChess
YAML config: https://github.com/eskopp/AdventofChess/blob/8f7beae409f36b1ee49eb59f3f416f6dbd92a950/.github/workflows/ilmenauersv.yml
YAML workflow: https://github.com/eskopp/AdventofChess/actions/runs/12333833779/job/34423278132
```

### Relevant log output

```
Run peaceiris/actions-hugo@v3
  with:
    hugo-version: latest
    extended: true
  env:
    DATETIME: 2024_[1](https://github.com/eskopp/AdventofChess/actions/runs/12333833779/job/34423278132#step:7:1)2_14_23_11
    pythonLocation: /opt/hostedtoolcache/Python/3.1[2](https://github.com/eskopp/AdventofChess/actions/runs/12333833779/job/34423278132#step:7:2).8/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/[3](https://github.com/eskopp/AdventofChess/actions/runs/12333833779/job/34423278132#step:7:3).12.8/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.8/x6[4](https://github.com/eskopp/AdventofChess/actions/runs/12333833779/job/34423278132#step:7:4)
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.8/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.8/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.8/x64/lib
Hugo version: 0.139.[5](https://github.com/eskopp/AdventofChess/actions/runs/12333833779/job/34423278132#step:7:5)
Error: Action failed with error Unexpected HTTP response: 404
```

### Additional context.

I resolved the issue by simply using an older version of Hugo. It would be great if the latest command worked again in the future! :)

```yml
      - name: Install Hugo Extended
        uses: peaceiris/actions-hugo@v3
        with:
          hugo-version: "0.139.2"
          extended: true
```
See: [Hugo Versions](https://github.com/gohugoio/hugo/releases)

**Comment by peaceiris** [comment]
Thank you for the report! The cause seems to be the issue mentioned below.
Considering a fallback for such patterns might be a good idea, but for now, it is recommended to specify a particular version instead of using latest.

- https://github.com/peaceiris/actions-hugo/issues/652#issuecomment-2543985304

**Comment by AnimMouse** [comment]
Now fixed with v0.140.0 release.

**Comment by eskopp** [comment]
> Now fixed with v0.140.0 release.

ok. thanks

**Comment by peaceiris** [comment]
Thanks all.
